packages/react/focus-scope/src/FocusScope.tsx

```diff
@@ -22,10 +22,16 @@ type FocusScopeOwnProps = Polymorphic.Merge<
   {
     /**
      * Whether focus should be trapped within the FocusScope
-     * (default: false)
+     * @defaultValue false
      */
     trapped?: boolean;
 
+    /**
+     * The element to auto focus when unmounted.
+     * @defaultValue false
+     */
+    unmountAutoFocusElement?: HTMLElement | null;
```

Comment:
I'm not attached to this name so feel free to suggest alternatives 😛

Reply:
Honestly can't think of a better one either 😅